src/Sylius/Bundle/ApiBundle/Resources/config/serialization/PaymentMethod.xml

```diff
@@ -20,25 +20,58 @@
             <group>admin:payment_method:read</group>
             <group>shop:payment_method:read</group>
         </attribute>
-        <attribute name="name">
-            <group>admin:payment:read</group>
+
+        <attribute name="translations">
```

Comment:
Just in case, is it intentional to expose `translations` and `name`, `description`, `instructions` for `shop:payment_method:read` at the same time?

Reply:
Good catch, it shouldn't be exposed